src/components/Counter/Counter.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { Input } from 'wix-ui-core/input';
+import { Button } from 'wix-ui-core/button';
+import { ReactComponent as PlusIcon } from '../../assets/icons/plus.svg';
+import { ReactComponent as MinusIcon } from '../../assets/icons/minus.svg';
+import style from './Counter.st.css';
+
+export interface CounterProps {
+  onChange(val: string): void;
+
+  value: number;
+  step: number;
+  min: number;
+  max?: number;
+  error?: boolean;
+  disabled?: boolean;
+}
+
+interface DefaultProps {
+  step: number;
+  min: number;
+  value: number;
+}
+
+/** Counter */
+export class Counter extends React.Component<CounterProps> {
+  static displayName = 'Counter';
+  static defaultProps: DefaultProps = {
+    step: 1,
+    min: 0,
+    value: 0,
+  };
+
+  onDecrement = () => {
+    const { step, max, onChange, value } = this.props;
+    if (max) {
+      const result = value > max ? max : value - step;
+      onChange(result.toString());
+      return;
+    }
+    onChange((value - step).toString());
+  };
+
+  onIncrement = () => {
+    const { step, min, onChange, value } = this.props;
+    const result = value < min ? min : value + step;
+
+    onChange(result.toString());
+  };
+
+  render() {
+    const { min, max, step, disabled, onChange, value, error } = this.props;
+
+    return (
+      <TPAComponentsConsumer>
```

Comment:
Currently there's no special `mobile` design for this component, so all references for `mobile` can be removed.
So `<TPAComponentsConsumer>` and `<TPAComponentsProvider>` can be removed, and all tests that test `mobile` state.
It's something that is automatically generated, and we changed it now, so that there won't be too much "noise"

Reply:
done